e2e/claims.service.e2e.ts

```diff
@@ -321,8 +397,8 @@ describe('Enrollment claim tests', () => {
         returnSteps: false,
       });
 
-      mockGetClaimsBySubject.mockImplementationOnce(() => []);
-      mockGetClaimsBySubject.mockImplementationOnce(() => []);
+      mockGetClaimsBySubject.mockImplementation(() => []);
```

Comment:
Are we making sure to restore this mocked value since we don't mock it once?

Reply:
I fixed tests to mockImplementationOnce everywhere